e2e/Forc.toml

```diff
@@ -118,3 +118,6 @@ members = [
   'sway/types/scripts/script_u256',
   'sway/types/scripts/script_vectors',
 ]
+
+[patch.'https://github.com/fuellabs/sway']
+std = { git = "https://github.com/fuellabs/sway", branch = "ironcev/error-codes-in-abi-json" }
```

Comment:
This also feels like something that we should clean up before merging.

Reply:
see above